bylaws/7-indemnification-dissolution.md

```diff
@@ -22,54 +22,71 @@ The CF Council (CFC) shall assess the Foundation's risks and evaluate whether to
 
-Subject to the limitations set forth in this Section,
-the Foundation must indemnify directors (councilors), Officers, employees, and agents of the Foundation
-from any action, suit, or proceeding (whether civil, criminal, administrative or investigative)
-arising from that person’s role in the Foundation or service at the request of the Foundation.
+Subject to the limitations set forth in this Section, the Foundation shall, to the fullest extent permitted by applicable law, indemnify and hold harmless any person (a “Covered Person”) who is made a party to, or threatened to be made a party to, any action, suit, or proceeding, whether civil, criminal, administrative, or investigative (other than an action by or in the right of the Foundation) (a “Proceeding”), by reason of the fact that they are or were a council member (acting as a director) or officer of the Foundation, against all liability and loss suffered and expenses (including attorneys' fees) reasonably incurred by that person.
+
+The Foundation may, at the discretion of the CF Council (CFC), extend indemnification to employees, committee members, and other agents of the Foundation as determined appropriate by the CFC.
+
+> [!TIP]
+> This means the Foundation will cover costs and losses for council members and officers who are sued because of their role in the Foundation, as long as they acted honestly and in the Foundation's best interest.
+>
+> The council can choose to extend this coverage to staff and committee members if they decide it’s appropriate.
 
 **Standard of Conduct.**
-This indemnification extends to all reasonable expenses (including attorneys’ fees), judgments, fines, and other amounts paid in settlement by the person in connection with the action, so long as the person:
-(1) acted in good faith and in a manner they reasonably believed to be consistent with the best interests of the Foundation, and
-(2) had no reason to believe their conduct was unlawful, in the case of criminal matters.
+
+This indemnification extends to all reasonable expenses (like legal fees), judgments, fines, and settlement amounts that a person may have to pay in connection with a legal action, so long as the person:
+
+1. acted in good faith and in a manner they reasonably believed to be consistent with the best interests of the Foundation, and
+2. had no reason to believe their conduct was unlawful, in the case of criminal matters.
 
 This indemnification does not extend to a person who is found liable to the Foundation in an action brought by the Foundation, or to enforce the Foundation's rights.
 Otherwise, the outcome of an action will not alone create a presumption that a person did not act in good faith, in the best interests of the Foundation, or with lawful intent.
 
+> [!TIP]
+> If the Foundation brings a lawsuit against a person, or if the person is found responsible for wrongdoing in such a lawsuit, they will not be covered by this indemnification.
```

Comment:
Hum, we are back to "they". They is the person? Also, do it applies to legal entities or only to physical entities?

Reply:
Commonhaus is indemnifying people (as an organization, our members and representatives are people), not companies or other legal entities.